examples/README.md

```diff
@@ -104,6 +110,25 @@ wandb.login()
 
 Whenever you use `Trainer` or `TFTrainer` classes, your losses, evaluation metrics, model topology and gradients (for `Trainer` only) will automatically be logged.
 
-For advanced configuration and examples, refer to the [W&B documentation](https://docs.wandb.com/library/integrations/huggingface).
-
 When using 🤗 Transformers with PyTorch Lightning, runs can be tracked through `WandbLogger`. Refer to related [documentation & examples](https://docs.wandb.com/library/frameworks/pytorch/lightning).
+
+### Comet.ml
+
+To use `comet_ml`, install the Python package with:
+
+```bash
+pip install comet_ml
+```
+
+or if in a Conda environment:
+
+```bash
+conda install -c comet_ml -c anaconda -c conda-forge comet_ml
+```
+
+### Further Reading
+
+For advanced configuration and examples, refer one of the following options:
+
+* [W&B documentation](https://docs.wandb.com/library/integrations/huggingface)
```

Comment:
It may be cleaner when the advanced usage is part of respective library sections. For example here it answers many common questions related to the logger. What do you think @sgugger?

Reply:
Reading further, the same link is given in the table of contents above (line 87) so I'd remove this section entirely since it doesn't bring anything new.

Reply:
Will do. I didn't want to remove anything in the first pass, but I agree. I'll fix.

Reply:
Done!